tox.ini

```diff
@@ -45,9 +48,12 @@ deps =
     django32: Django>=3.2,<3.3
     django41: Django>=4.1,<4.2
```

Comment:
would be good to drop Django < 4.2 and Wagtail < 5.2

Reply:
That sound sensible, for clarity are you suggesting the following:
```ini
# File: tox.ini
##########

envlist =
    py{38,39,310,311,312}-django{42,50}-wagtail{52,60}
    interactive


...
    django42: Django>=4.2,<5.0
    django50: Django>=5.0,<5.1
    wagtail52: wagtail>=5.2,<6.0
    wagtail60: wagtail>=6.0,<6.1
    interactive: wagtail>=5.2
```
Note: I referenced https://docs.wagtail.org/en/stable/releases/upgrading.html#compatible-django-python-versions to decide on the envlist version mapping.

Curious to understand how support is decided for Wagtail though, it looks like Django has dropped long term support for 4.1 in December, however, how do we decide on what versions of Wagtail to support?
Wouldn't it make more sense to support Wagtail < 5.0 since it's compatible with Django 4.2?

Would we also want to suppor Python 3.7?

Reply:
What versions one supports is a mixed bag of
- guidance from https://github.com/wagtail-nest/
- officially supported versions
  - https://endoflife.date/python (we can see clearly that 3.7 is out of the picture, and frankly we should think of starting to drop 3.8 too)
  - https://endoflife.date/django
  - https://endoflife.date/wagtail

in an ideal world we'd support the widest range, but then that adds maintenance burder, increases the CI time and resource waste, and doesn't help push implementers to keep up to date.

I talked about my approach at our internal tech conference in October

Reply:
Nice, thanks for the response Dan, I'll also see about watching your talk, I was ill that day but have a link to the recordings.

I've made a slight change to the orginal envlist since my inital comment, as it looks like Django doesn't support python 3.8 and 3.9 https://docs.djangoproject.com/en/5.0/faq/install/#what-python-version-can-i-use-with-django

Otherwise I think this should be fine now, sorry I've made a mess of the commits trying to fix CI errors, I can spend time cleaning them up if that will help with reviewing.